packages/wallet-service/src/api/addresses.ts

```diff
@@ -125,7 +125,7 @@ export const get: APIGatewayProxyHandler = middy(
       return closeDbAndGetError(mysql, ApiError.WALLET_NOT_READY);
     }
 
-    const { value: body, error } = AddressAtIndexValidator.validate(event.pathParameters);
+    const { value: body, error } = AddressAtIndexValidator.validate(event.pathParameters || {});
```

Comment:
`event.pathParameters` used to return an empty object when nothing was sent, this changed after we updated serverless